assets/js/blocks/customer-account/frontend.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { renderFrontend } from '@woocommerce/base-utils';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+
+export const parseAttributes = ( data: Record< string, unknown > ) => {
+	return {
+		displayStyle: data?.displayStyle,
+		iconStyle: data?.iconStyle,
+	};
+};
+
+const getProps = ( el: HTMLElement ) => {
+	return {
+		attributes: parseAttributes( el.dataset ),
+	};
+};
+
+renderFrontend( {
+	selector: '.wp-block-woocommerce-customer-account',
+	Block,
+	getProps,
+} );
```

Comment:
For now, I don't think the block should have any client-side (JS) rendering as it means we're loading all of React (and related dependencies) unnecessarily. Let's move this to server-side rendering.

Reply:
Makes sense, thanks Darren. I just pushed the change :) Can you have another look?